contrib/doc/cjdnslog.md

```diff
@@ -0,0 +1,43 @@
+cjdnslog(1) -- display cjdroute log messages
+=============================================
+
+## SYNOPSIS
+
+`/usr/bin/cjdnslog` [<options>...]
+
+## DESCRIPTION
+
+Cjdroute sends continuous log messages to the bit bucket.  Cjdnslog
```

Comment:
teeeechnically the logging never happens unless you subscribe/request the log lines, but however you care to word it is probably fine.